CardanoSharp.Wallet.Test/TransactionTests.cs

```diff
@@ -155,25 +305,173 @@ public void DeserializeMultiAssetTransaction()
         }
 
         [Fact]
-        public void SerializeTransaction()
```

Comment:
SerializeTranasaction is below. 

I added the DeserializeMultiAssetTransactionValues function for value comparision